packages/commonwealth/client/styles/SublayoutHeader.scss

```diff
@@ -70,6 +70,10 @@ html.todesktop .SublayoutHeader {
       @include extraSmall {
         display: flex;
       }
+
+      @include smallInclusive {
```

Comment:
I dont think this is desired behaviour @Israellund. Menu has two types
1. desktop => icons visible in top navbar
2. mobile => extendable 3 dots menu

As a user you should be able to see one or the other, not both in the same time. So at a breakpoint when you are showing 3 dots menu, in the same time, you should hide `DesktopMenuContainer`.